python/lsst/galaxy_shear/processShearTest.py

```diff
@@ -172,7 +235,29 @@ def run(self, dataRef):
             except Exception as error:
                 self.log.warn("Error in %s.measure on record %s: %s"
                               % (self.measurement.plugins[plugin].name, source.getId(), error))
-        dataRef.put(sourceCat, "src")
+            if not self.e1Key is None:
+                e1 = source.get(self.e1Key)
+                e2 = source.get(self.e2Key)
+            else:
+                xx = source.get(self.xxKey)
+                yy = source.get(self.yyKey)
+                xy = source.get(self.xyKey)
+                e1 = (xx - yy) / (xx + yy)
+                e2 = 2 * xy / (xx + yy)
+            e = math.sqrt(e1 * e1 + e2 * e2)
+
+            #   Calculate the rotation in the range [-pi, pi], just to check the pairs
+            theta = 180.0*math.atan2(e2,e1)/(2.0*math.pi)
+            while theta < 0:
+                theta = 360 + theta
+            while theta > 90.0:
+                theta = theta-90.0
```

Comment:
I am using some of these calculations during debugging.  I am still not confident that the CModel calculation is working in all cases.  The angle calculation is useful when examining individual measurements, so I will leave it here until if it is no longer useful.